### PR TITLE
Flatfile hotfix

### PR DESCRIPTION
**Pull request by brmscheiner**

### CODE REVIEW

**Review by Atticus29 — APPROVED** [review]
Weird to me that this fixes anything. What was broken before?

File: src/pages/bulkImport/utils/useBulkImportFields.js [hunk 1]
```diff
@@ -32,6 +32,13 @@ const utcOffsetValidator = {
   error: 'Not a valid timezone (UTC offset format)',
 };
 
+function deriveLabel(field, intl) {
+  const label = field?.labelId
+    ? intl.formatMessage({ id: field.labelId })
+    : field.label;
```
Comment (Emily-Ke) [inline src/pages/bulkImport/utils/useBulkImportFields.js:38]: If `field?.labelId` can be null because field can be null, then this could still error.

Comment (brmscheiner) [reply src/pages/bulkImport/utils/useBulkImportFields.js]: thanks. that wasn't the original issue but still worth fixing.

**Comment by brmscheiner** [comment]
@Atticus29 there is a bad custom field creating by backend tests that is hanging out on the QA server. it has no schema and therefore no label. this attempts to make the bulk import UI continue to work in such cases

File: src/pages/bulkImport/utils/useBulkImportFields.js (hunk 1 above)
Comment (brmscheiner) [reply src/pages/bulkImport/utils/useBulkImportFields.js]: @Emily-Ke pushed

Comment (Atticus29) [reply src/pages/bulkImport/utils/useBulkImportFields.js]: As, in the bottom part should be:

`field?.label `
as well?

Or does:
`null ? something: somethingElse `
inherently cause an error?

Comment (brmscheiner) [reply src/pages/bulkImport/utils/useBulkImportFields.js]: the first one! `?.`

Comment (Atticus29) [reply src/pages/bulkImport/utils/useBulkImportFields.js]: I'm seeing from the new commit that it's the former.

**Review by Emily-Ke — APPROVED** [review]

**Pull request merged**